azext_iot/central/params.py

```diff
@@ -214,12 +237,152 @@ def load_central_arguments(self, _):
         context.argument(
             "module_id",
             options_list=["--module-id", "-m"],
-            help="Provide IoT Edge Module ID if the device type is IoT Edge.",
+            help="The IoT Edge Module ID if the device type is IoT Edge.",
         )
 
     with self.argument_context("iot central role") as context:
         context.argument(
             "role_id",
             options_list=["--role-id", "-r"],
-            help="Provide a unique identifier for the role",
+            help="Unique identifier for the role",
+        )
+
+    with self.argument_context("iot central file-upload-configuration") as context:
+        context.argument(
+            "api_version",
+            options_list=["--api-version", "--av"],
+            choices=CaseInsensitiveList([ApiVersion.v1_1_preview.value]),
+            default=ApiVersion.v1_1_preview.value,
+            help="The API version for the requested operation.",
+        )
+
+    with self.argument_context(
+        "iot central file-upload-configuration create"
+    ) as context:
+        context.argument(
+            "connection_string",
+            options_list=["--connection-string", "-s"],
+            help="The connection string used to configure the storage account",
+        )
+        context.argument(
+            "container",
+            options_list=["--container", "-c"],
+            help="The name of the container inside the storage account",
+        )
+        context.argument(
+            "account",
+            options_list=["--account", "-a"],
+            help="The storage account name where to upload the file to",
+        )
+        context.argument(
+            "sasTtl",
+            options_list=["--sas-ttl"],
+            help="The amount of time the device’s request to upload a file is valid before it expires."
+            " ISO 8601 duration standard. Default 1h.",
+        )
+
+    with self.argument_context("iot central organization") as context:
+        context.argument(
+            "org_id",
+            options_list=["--org-id"],
+            help="Unique identifier for the organization.",
+        )
+        context.argument(
+            "api_version",
+            options_list=["--api-version", "--av"],
```

Comment:
Try and de-dupe the parameters as much as possible.

Assuming `--api-version` being defined multiple times for different command groups is due to inconsistent API version support in central i.e. ApiVersion.v1_1_preview and ApiVersion.preview support for `az iot central job` vs only ApiVersion.v1_1_preview support in `az iot central organization`.

Reply:
it was actually meant in that way. orgs are only available for 1.1-preview. Same for others.